src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraEntity.scala

```diff
@@ -25,7 +38,8 @@ case class AgoraEntity(@(ApiModelProperty@field)(required = false, value = "The
                        @(ApiModelProperty@field)(required = false, value = "The method payload")
                        payload: Option[String] = None,
                        @(ApiModelProperty@field)(required = false, value = "URI for method details")
-                       url: Option[String] = None
+                       url: Option[String] = None,
+                       entityType: Option[AgoraEntityType.EntityType]= None
```

Comment:
We are packing entityType into metadata? I thought it was just being encoded by storing it in different underlying collections and being asked-for by different endpoints. If we do want it in the metadata packet for an Entity, should it have a swagger ApiModelProperty?